fdbserver/ptxn/test/Driver.actor.cpp

```diff
@@ -214,6 +214,242 @@ void startFakeStorageServer(std::vector<Future<Void>>& actors, std::shared_ptr<T
 	}
 }
 
+namespace details {
+
+#pragma region TLogGroupFixture
```

Comment:
It seems each of these `#pragma region` is just a function. The editor is perfectly able to handle without them. Maybe remove them (or at least those just wrapping one function)?